TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+and enforcement of the Knative mark. It is a separate body from the
+[Steering Committee](./STEERING-COMMITTEE.md) to better represent the needs of
+vendors and other companies which which to market products in conjuction with
+the Knative mark.
+
+The governance of Knative is an open, living document, and will continue to
+evolve as the community and project change. We expect over time we will adapt
+the way we run this committee, based on feedback from the community.
+
+- [Charter](#charter)
+- [Delegated Authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision Process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Composition](#composition)
+  - [Maintaining the Trademark Committee](#maintaining-the-trademark-committee)
+
+## Charter
+
+1. Advise the Trademark owner on the definition and enforcement of the "Knative"
+   trademark usage and branding guidelines.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve the scope and definition of additional derivative Knative
+   marks (e.g. Knative Serving, Knative Core, etc). It is expected that the
+   Trademark Committee will delegate one or more marks to the steering committee
+   to provide branding for nascent efforts which are not yet subject to
+   conformance and third-party licensing.
+
+1. Only the Trademark Committee can redefine its own charter, membership, and
+   authority over Knative Trademark, conformance, and brand decisions.
+
+## Delegated authority
+
+The Trademark Committee may choose create and delegate its authority to other
+committees as-needed. This may be used (for example) to create sub-committees
+for specific technical or marketing questions related to conformance or joint
+marketing.
+
+## Committee Meetings
+
+The committee meets as-needed. Meetings are held online.
+
+Given the private nature of some of these discussions (e.g. trade secrets, legal
+and compliance concerns, etc.) some meetings are held in private.
+
+Meeting notes are available to members of the
+[knative-dev mailing list](https://groups.google.com/forum/#!forum/knative-dev)
+(note: these are accessible to members of the knative-users list as well).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as
+[issues in the community repo](https://github.com/knative/community/issues), and
+the trademark committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee Members
+
+| Member                   | Organization | Profile |
+| ------------------------ | ------------ | ------- |
+| Unknown - to be selected | Google       | Unknown |
+| Unknown - to be selected | IBM/RedHat   | Unknown |
+| Unknown - to be selected | VMware       | Unknown |
+
```

Comment:
We should get the vendors to fill these in before committing the PR. Having said that I don't know who it'll be for Google, and IBM/RedHat also need to make a choice; for VMWare we can rely on Brenda to let us know if she'll serve for a year or pass the baton.

Reply:
If we follow the proposed process, I think Google/IBM/VMware would each put forward a name and the current Steering would approve them.

If we need to get that done before landing the PR, then we need to get names ASAP.

Alternatively, we can put in names, and their company can replace them 2 days later if needed, according to these rules as writ. I think things are written such that it should be fairly harmless to put names in later, but I defer to the experts here.

Reply:
An alternative approach is that the trademark committee forms when the newly elected steering committee is formed so that the transition period stays the same for the project. That way vendors have a month to decide who should be on the trademark committee.